src/aggregation/mod.rs

```diff
@@ -189,6 +199,15 @@ impl<T: Clone> From<HashMap<String, T>> for VecWithNames<T> {
 }
 
 impl<T: Clone> VecWithNames<T> {
+    fn from_other<K: Clone + Into<T>>(entries: VecWithNames<K>) -> Self {
+        let mut values = Vec::with_capacity(entries.len());
+        values.extend(entries.values.into_iter().map(Into::into));
```

Comment:
!??

`let values = entries.values.into_iter().map(Into::into).collect();`

Reply:
that was a small performance optimization, but I don't think we need it